node/src/actors/chain_manager/handlers.rs

```diff
@@ -210,6 +210,15 @@ impl Handler<EpochNotification<EveryEpochPayload>> for ChainManager {
             }
         }
 
+        // After block consolidation, commits and reveals that arrive in an incorrect moment
+        // are processed now
+        let timestamp_now = get_timestamp();
+        let delayed_transactions: Vec<Transaction> =
+            self.temp_commits_and_reveals.drain(..).collect();
+        for transaction in delayed_transactions {
+            let _aux = self.add_transaction(AddTransaction { transaction }, timestamp_now);
```

Comment:
is _aux needed?

Reply:
clippy requirements

Reply:
`add_transaction` returns a future, so it will not be executed... Try `ctx.notify(AddTransaction` instead

Reply:
done!